solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1052,7 +1068,39 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
 
         require(averageRateInRange(spotRateN, spotRateD, averageRateN, averageRateD, averageRateMaxDeviation), "ERR_INVALID_RATE");
 
-        return Fraction(averageRateN, averageRateD);
+        return (spotRateN, spotRateD, averageRateN, averageRateD);
+    }
+
+    /**
+      * @dev returns a packed object containing the reserves rates
+      *
+      * @param _poolToken       pool token
+      * @param _reserveToken    reserve token
+      * @param _addSpotRateN    add spot rate numerator
+      * @param _addSpotRateD    add spot rate denominator
+      * @return packed object containing the reserves rates
+    */
+    function getPackedRates(
```

Comment:
`get` again

Reply:
Desired name please (function and variable, where the function is called).

Reply:
figure it out according to our standards

Reply:
Fixed.